File: back-end/tea-with-milk/src/main/java/com/pwy/entity/pojo/Shop.java
package com.pwy.entity.pojo;

import lombok.Data;

//店铺实体类
@Data
public class Shop {
    private String id;

    private String name;

    private String longitude;

    private String latitude;

    private String position;
    //账号
    private String account;
    //密码
    private String password;

    private String phone;

    private String logo;
}

File: front-end/tea-with-milk/src/apis/Shop.ts
import type { ShopInfo } from "@/types/Shop"
import http from "@/utils/http"

export const getShopInfoAPI=()=>{
  return http<ShopInfo>({
    url:'/shop/info',
    method:'GET'
  })
}
//修改门店信息
export const updateShopInfoAPI=(data:{name:string,longitude:number,latitude:number,position:string,phone:string})=>{
  return http({
    url:'/shop/update',
    method:'PUT',
    data
  })

}
//修改店铺密码
export const updateShopPasswordAPI=(data:{originalPassword:string,newPassword:string})=>{
  return http({
    url:'/shop/password',
    method:'PUT',
    data
  })
}